### PR TITLE
Make iron-test build with stable Rust

### PR DESCRIPTION
**Pull request by upsuper**

### CODE REVIEW

File: src/project_builder.rs [hunk 1]
```diff
@@ -46,11 +44,12 @@ impl FileBuilder {
 ///
 /// It is also a builder and is used to build up the temporary files,
 /// which are then deleted on drop.
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, Clone)]
 pub struct ProjectBuilder {
     name: String,
     root: PathBuf,
     files: Vec<FileBuilder>,
+    lock: Arc<Mutex<u32>>,
```
Comment (reem) [inline src/project_builder.rs]: it looks like the data here is unused, so `Arc<Mutex<()>>` would be more appropriate

**Comment by reem** [comment]
Looks good except for the data in the lock.

File: src/project_builder.rs (hunk 1 above)
Comment (upsuper) [inline src/project_builder.rs]: Fixed in  57eb156

**Comment by reem** [comment]
Merged and release 0.1.0

**Pull request merged**